fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/schema/SchemaVersionsManager.java

```diff
@@ -90,15 +88,28 @@ public void updateSchemaVersionId(int versionId) {
      * table
      */
     public void updateSchemaVersion() {
-        updateSchemaVersionId(this.latestCodeVersion);
+        // The schema version should never regress, so make sure we don't change
+        // anything if this code is older than what's currently in the database
+        if (latestCodeVersion > getVersionForSchema()) {
+            updateSchemaVersionId(this.latestCodeVersion);
+        }
+    }
+
+    /**
+     * Returns true if the current schema version recorded in WHOLE_SCHEMA_VERSION is older
+     * the last version in FhirSchemaVersion and therefore needs to be updated
```

Comment:
super minor, only update if making other edits
```suggestion
     * than the last version in FhirSchemaVersion and therefore needs to be updated
```